lib/matplotlib/colors.py

```diff
@@ -1061,11 +1061,11 @@ def scaled(self):
 
 
 class DivergingNorm(Normalize):
-    def __init__(self, vcenter, vmin=None, vmax=None):
+    def __init__(self, vcenter, vmin=None, vmax=None, fair=False):
```

Comment:
I'll block on this being kwarg.  Skptical about the normalization on the whole, but very against it being a kwarg for `DivergingNorm`

Reply:
But I think this belongs in DivergingNorm, so can we pitch this to the call? (which I'm probably going to be late to).

Reply:
From a user point of view `norm1 = OffcenterNorm(0.0, vmin=-0.25, vmax=0.75)` is easier to type, and can get its own section in the docs, rather than making the docs and code for `DivergingNorm` have a confusing kwarg that does something quite different from what `fair=False` does.  I don't understand the desire to make this a kwarg.